arango/database.py

```diff
@@ -838,7 +838,7 @@ def metrics(self) -> Result[str]:
         :return: Server metrics in Prometheus format.
         :rtype: str
         """
-        request = Request(method="get", endpoint="/_admin/metrics")
+        request = Request(method="get", endpoint="/_admin/metrics/v2")
```

Comment:
todo: emit a warning if DB Version is <3.10

Reply:
added in [de92c09](https://github.com/ArangoDB-Community/python-arango/pull/313/commits/de92c0943df07956cd1a97d90814bb22f149118b), see comment below